pyretailscience/plots/styles/styling_helpers.py

```diff
@@ -0,0 +1,121 @@
+"""PlotStyler class that applies styling using the context."""
+
+from matplotlib.axes import Axes
+from matplotlib.text import Text
+
+from pyretailscience.plots.styles.styling_context import get_styling_context
+from pyretailscience.plots.styles.tailwind import get_multi_color_cmap, get_single_color_cmap
+
+
+class PlotStyler:
+    """Helper class for applying all styling - fonts, colors, and hardcoded elements."""
+
+    def __init__(self) -> None:
+        """Initialize the PlotStyler with the current styling context."""
+        self.context = get_styling_context()
+
+    def apply_base_styling(self, ax: Axes) -> None:
+        """Apply base plot styling (spines, grid, background) - using hardcoded defaults."""
+        # These remain hardcoded as they represent the pyretailscience visual identity
+        ax.set_facecolor("w")
+        ax.set_axisbelow(True)
+        ax.spines[["top", "right"]].set_visible(False)
+        ax.grid(which="major", axis="x", color="#DAD8D7", alpha=0.5, zorder=1)
+        ax.grid(which="major", axis="y", color="#DAD8D7", alpha=0.5, zorder=1)
+
+    def apply_title(self, ax: Axes, title: str, pad: int | None = None) -> None:
+        """Apply title styling using context fonts."""
+        fonts = self.context.fonts
+        pad = pad or 10
```

Comment:
This could be simplified.
```suggestion
    def apply_title(self, ax: Axes, title: str, pad: int | None = 10) -> None:
        """Apply title styling using context fonts."""
        fonts = self.context.fonts
```